gui/src/main/java/net/tudelft/hi/e/gui/MainUI.java

```
@@ -431,11 +436,44 @@ private void onRunScript() {
     }
     ProgressGui.init();
   }
+  
+  private void readPreviousDirectory() {
+    File prevDirectoryFile = new File("prev_directory.txt");
+    
+    if (!(prevDirectoryFile.exists() && prevDirectoryFile.isFile())) {
+      return;
+    }
+    try {
+      BufferedReader reader = new BufferedReader(new FileReader(prevDirectoryFile));
+      String line = reader.readLine();
+      File prevDirectory = new File(line);
+      
+      if (prevDirectory.exists() && prevDirectory.isDirectory()) {
+        previousDirectory = prevDirectory;
+      }
+      reader.close();
+    } catch (IOException e) {      
+    }
+  }
+  
+  private void writePreviousDirectory() {
+    if (previousDirectory == null) {
+      return;
+    }
+    
+    try {
+      FileWriter writer = new FileWriter("prev_directory.txt");
+      writer.write(previousDirectory.getAbsolutePath());
+      writer.close();
+    } catch (IOException e) {
+    }
```

Empty catch is a nogo, use a logger